sections/semantics-embedded-content.include

```diff
@@ -6122,10 +6122,10 @@ zero or more <{track}> elements, then
   var videoSection = document.getElementById('video');
   var videoElement = document.createElement('video');
   var support = videoElement.canPlayType('video/x-new-fictional-format;codecs="kittens,bunnies"');
-  if (support != "probably" &amp;&amp; "New Fictional Video Plugin" in navigator.plugins) {
+  if (support != "probably") {
```

Comment:
Hmm... Good point. OK. Here's my new thinking:

Without the plugins API, browsers will no longer be reporting on the state of any installed plugins--so, the web author will no longer be able to reason about the difference between a "maybe" response and a "probably" response. I agree that in most cases, we want to favor trying a native <video> tag over a plugin, so instead, I'll change this code in my next update to check for the empty string case, and in that case, try the plugin API, for other cases (maybe/probably) give the native video tag support a try. This will also eliminate the failure case, so that at least a video tag will be presented to the user--it just might not render anything. 

(I don't think there's a great solution here, and canPlayType is already very ambiguous as a test...)